ps_customtext.php

```diff
@@ -205,14 +205,25 @@ public function processSaveCustomText()
 
         foreach ($languages as $lang) {
             $text[$lang['id_lang']] = (string) Tools::getValue('text_' . $lang['id_lang']);
+
+            if (!Configuration::get('PS_ALLOW_HTML_IFRAME') && preg_match('/<iframe.*src=\"(.*)\".*><\/iframe>/isU', $text[$lang['id_lang']])) {
+                $this->context->controller->errors[] = $this->trans('If you want to use <iframe> inside your content, enable them in Preferences -> General -> Allow iframes on HTML fields', [], 'Admin.Notifications.Error');
```

Comment:
```suggestion
                $this->context->controller->errors[] = $this->trans('To use <iframe>, enable the feature in Shop Parameters> General', [], 'Admin.Notifications.Error');
```

Reply:
> Hello @Julievrz
> 
> Do you think we could have this newly introduced phrase in the global domain? I can imagine using it outside of the `ps_customtext` module too.

Yes, you can use 'Admin.Notifications.Error' :-)

Reply:
@Julievrz I added a space before `>`, unless you did that on purpose?

Reply:
Nope, it wasn't on purpose. Thanks, @kpodemski!